Tests/BraveTalkTests/BraveTalkTests.swift

```diff
@@ -9,9 +9,14 @@ import Shared
 import XCTest
 
 class BraveTalkTests: XCTestCase {
-  @MainActor func testBraveTalkJitsiIntegrationDisabledOnRelease() {
+//  @MainActor func testBraveTalkJitsiIntegrationDisabledOnRelease() {
```

Comment:
Maybe remove commented part, when we change we should update this test regardless

Reply:
Fixed